### PR TITLE
docs(buildsystem): add week 5 7 8 reports

### PR DESCRIPTION
**Pull request by avinal**
## Project

Buildsystem

### Changes

- Add weekly reports for week 5 7 8
- Add more instructions to the build.md

### CODE REVIEW

File: docs/2021/buildsystem/updates/2021-07-09.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+-   [Sarita Singh](https://github.com/itssingh)
+-   [Avinal Kumar](https://github.com/avinal)
+
+### Week 5 Progress
+
+<ul><li>CMake Installation Configuration is almost complete.</li>
+    <li>FOSSology can be installed completely via CMake</li>
+    <li>Post install script generation also added</li>
+    <li>To test the current progress, follow the instructions <a href="https://github.com/avinal/FOSSology/wiki#test-the-new-system-only-gcc-with-make-and-ninja-tested-for-now">here</a></li>
+    </ul>
```
Comment (GMishx) [inline docs/2021/buildsystem/updates/2021-07-09.md:30]: Can be converted to MD lists? (using `*` or `-`)

File: docs/2021/buildsystem/updates/2021-07-23.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+-   [Sarita Singh](https://github.com/itssingh)
+-   [Avinal Kumar](https://github.com/avinal)
+
+### Week 7 Progress
+
+<ul><li>Initial CMake packaging configuration implemented.</li>
+    <li>Packages can be built according to the FOSSology previous packaging structure.</li>
+    <li>Copyright, ecc and keyword now builds faster.</li>
+    <li>To test the current progress, follow the instructions <a href="https://github.com/avinal/FOSSology/wiki#test-the-new-system-only-gcc-with-make-and-ninja-tested-for-now">here</a></li>
+    </ul>
```
Comment (GMishx) [inline docs/2021/buildsystem/updates/2021-07-23.md:35]: This can be converted too.

File: docs/2021/buildsystem/updates/2021-07-30.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+
+### Week 8 Progress
+
+<ul><li>CMake Packaging configuration almost completed.</li>
+    <li>Packages can be built according to the FOSSology previous packaging structure.</li>
+    <li>Initial testing configuration added.</li>
+    <li>Ninja build has been fixed.</li>
+    <li>To test the current progress, follow the instructions <a href="https://github.com/avinal/FOSSology/wiki#test-the-new-system-only-gcc-with-make-and-ninja-tested-for-now">here</a></li>
+    </ul>
```
Comment (GMishx) [inline docs/2021/buildsystem/updates/2021-07-30.md:33]: And here as well.

File: docs/2021/buildsystem/updates/2021-07-23.md (hunk 2 above)
Comment (avinal) [reply docs/2021/buildsystem/updates/2021-07-23.md]: Fixed

**Comment by avinal** [comment]
Hello Gaurav, I fixed the older files too.

**Pull request merged**